decidim-core/lib/tasks/decidim_metrics_tasks.rake

```diff
@@ -43,15 +43,16 @@ namespace :decidim do
         raise ArgumentError if day.blank?
 
         (Date.parse(day)..Time.zone.today).each do |d|
+          current_day = d.to_s
           Decidim::Organization.find_each do |organization|
             if metric
               m_manifest = Decidim.metrics_registry.for(metric)
-              puts "[#{organization.name}]: rebuilding metric [#{metric}] for day [#{d}]"
-              call_metric_job(m_manifest, organization, day)
+              puts "[#{organization.name}]: rebuilding metric [#{metric}] for day [#{current_day}]"
```

Comment:
maybe I would use a `Rails.logger.info` in favor of `puts`, don't you?

Reply:
This output is from a rake task. I think it is better feedback to have it in the console. Maybe we can have both? console and logger?

Reply:
Both of them seems good option.